lib/src/test/java/nl/knaw/dans/dansschema/DdmVersionChangesTest.java

```diff
@@ -0,0 +1,181 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.dansschema;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.xml.sax.SAXParseException;
+
+import static nl.knaw.dans.dansschema.XmlReader.readXmlString;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class DdmVersionChangesTest {
+    private static final String xsdDir = "src/main/resources/md/ddm/";
+    private static final String ddmNamespaceV1 = "http://easy.dans.knaw.nl/schemas/md/ddm/";
+    private static final String ddmNamespaceV2 = "http://schemas.dans.knaw.nl/dataset/ddm-v2/";
+    private static SchemaValidator ddmValidatorV2;
+    private static SchemaValidator ddmValidatorV1;
+
+    @BeforeAll
+    public static void initValidators() throws Exception{
+        ddmValidatorV2 = new SchemaValidator(xsdDir + "v2/ddm.xsd");
+        ddmValidatorV1 = new SchemaValidator(xsdDir + "v1/ddm.xsd");
+    }
+
+
+    @Test
+    public void xml_without_root_namespace_should_not_parse() {
+        assertThatThrownBy(() -> (readXmlString("<ddm:DDM/>")).normalize())
+            .isInstanceOf(SAXParseException.class)
+            .hasMessage("The prefix \"ddm\" for element \"ddm:DDM\" is not bound.");
+    }
+
+    @Test
+    public void v1_should_require_schemas_dans_knaw_nl_name_space_() throws Exception {
+        var result = ddmValidatorV1.validateString("<ddm:DDM  xmlns:ddm='http://schemas.dans.knaw.nl/dataset/ddm-v2/'/>");
+        assertThat(result).hasSize(1);
+        assertThat(result.get(0))
+            .hasMessage("cvc-elt.1.a: Cannot find the declaration of element 'ddm:DDM'.");
+    }
+
+    @Test
+    public void v2_should_require_easy_dans_knaw_nl_name_space() throws Exception {
+        var result = ddmValidatorV2.validateString("<ddm:DDM xmlns:ddm='http://easy.dans.knaw.nl/schemas/md/ddm/'/>");
+        assertThat(result).hasSize(1);
+        assertThat(result.get(0))
+            .hasMessage("cvc-elt.1.a: Cannot find the declaration of element 'ddm:DDM'.");
+    }
+
+    @Test
+    public void v1_should_allow_additional_xml_in_root() throws Exception {
+        var result = ddmValidatorV1.validateString("<ddm:DDM xmlns:ddm='http://easy.dans.knaw.nl/schemas/md/ddm/'/>");
+        assertThat(result).hasSize(1);
+        assertThat(result.get(0))
+            .hasMessageContaining("The content of element 'ddm:DDM' is not complete. One of")
+            .hasMessageContaining(":profile, ")
+            .hasMessageContaining(":dcmiMetadata, ")
+            .hasMessageContaining(":additional-xml}' is expected.");
+    }
+
+    @Test
+    public void v2_should_not_allow_additional_xml_in_root() throws Exception {
```

Comment:
Again, the name doesn't match what is tested. Even less, I should say, as additional-xml is not even mentioned in the error message.